rebar.config

```diff
@@ -38,7 +38,7 @@
   [{tx_xform, {git, "https://github.com/aeternity/tx_xform.git", {ref, "3e74c07"}}}]}.
```

Comment:
This will have to change (it seems it does not affect rebar.lock though).

Reply:
This will change when aeternity/tx_xform#4 is merged.